### ISSUE TITLE
tsocmd not working in zowe-support.sh

### ISSUE THREAD

**Issue by stevenhorsman** [zowe/zowe-install-packaging#1107]
In zowe-support it tries to capture the output of the zowe job by calling tsocmd output, but this doesn’t seem to work on my system. When I call tsocmd "output ZWES2SV(STC63193)" it guess prints the command `output ZWES2SV(STC63193)` and even worse seems to purge the job from the system.
The doc page (https://www.ibm.com/support/knowledgecenter/sk/SSLTBW_2.3.0/com.ibm.zos.v2r3.bpxa500/tsonew.htm) states ’Do not issue 'tsocmd output' from the OMVS shell. If you do, you might not receive any output.' so maybe it is bound to fail, so does anyone have any suggestions of how else we can capture the job info into the support pax, or spot what we are doing wrong in zowe-support.sh?

We could consider using z/OSMF's API to get it, but I don’t think we can add that dependency to the zowe-support.sh, especially if z/OSMF, or configuration is the problem and it means we’d have to ship another script (either node, or Java) to make that possible

**Comment by John-A-Davies** [comment]
The script `zowe-support.sh` uses the STATUS command to obtain the JOBID to be printed, but the format of the STATUS command is different on some systems.
```
TSTRADM:/u/tstradm: >tsocmd status zwe1sv
status zwe1sv
IKJ56211I JOB ZWE1SV(STC01637) EXECUTING
IKJ56192I JOB ZWE1SV(STC01209) ON OUTPUT QUEUE
IKJ56192I JOB ZWE1SV(STC01230) ON OUTPUT QUEUE
```
and
```
/u/tstradm:>tsocmd status zwe1sv
status zwe1sv
JOB ZWE1SV(STC64897) EXECUTING
JOB ZWE1SV(STC64915) EXECUTING
```
so the script needs to tolerate that.

**Comment by John-A-Davies** [comment]
If you use
```
output jobname(jobid) print(dsn)
```
then the output will be placed in DSN=userid.dsn.OUTLIST and the job will be purged.

**Comment by John-A-Davies** [comment]
The script zowe-support.sh amended by this fix will tolerate different formats of the output of the STATUS command.

**Comment by John-A-Davies** [comment]
The fix for this defect will enable the TSO OUTPUT command to be launched from the `zowe-support.sh` script, so that Zowe JES job logs can be collected.

However, sites with an unmodified version of z/OS may restrict the OUTPUT command to jobs starting with your userid.  This happens, for example, on unmodifed ZD&T systems.  When it happens, the `zowe-support.sh` script will display the message
```
 IKJ56328I JOB jobname REJECTED - JOBNAME MUST BE YOUR USERID OR MUST START WITH YOUR USERID
```
Refer to IBM documentation of the IKJ56328I message for an explanation.

To work around this restriction, you can print the desired JES spool log by one of these methods:

1. Use an amended IKJEFF53 exit, together with related spool access settings
2. Use SDSF if available; see https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zosmfincidentlog.help.doc/izuP00hpActCopyJLtoDS.html for instructions
3. Use another spool manager product if SDSF is not available

**Comment by John-A-Davies** [comment]
For TSO OUTPUT to collect the job logs you want, refer to https://www.ibm.com/support/knowledgecenter/SSLTBW_2.2.0/com.ibm.zos.v2r2.e0ze100/ikjeff53.htm

**Comment by John-A-Davies** [comment]
Fixed by PR #1192
